test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -458,11 +458,6 @@ struct NUMASwitcher {
 #ifndef _AIX // JDK-8257041
 TEST_VM(os, release_multi_mappings) {
 
-  // With NMT enabled, this will trigger JDK-8263464. For now disable the test if NMT=on.
-  if (MemTracker::tracking_level() > NMT_off) {
-    return;
-  }
-
```

Comment:
Since appearantly it is possible to release part of a range, we need also a test that tests multi-region-release but with the last region only partly covered.

Reply:
Added `partial_release_multi_mappings` test to cover this.